src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -272,5 +339,38 @@ describe('ReviewScreen', () => {
         },
       })
     })
+    describe.each([
+      [
+        FiatAccountType.BankAccount,
+        FiatAccountSchema.AccountNumber,
+        'fiatConnectReviewScreen.bankFeeDisclaimer',
+      ],
+      [
+        FiatAccountType.MobileMoney,
+        FiatAccountSchema.MobileMoney,
+        'fiatConnectReviewScreen.mobileMoneyFeeDisclaimer',
+      ],
+    ])('Fee Disclaimer for %s', (accountType, schema, disclaimer) => {
```

Comment:
This test doesn't seem like it adds much value. Its essentially testing that for every country in the list the default currency is not USD. Thats testing the `country-data` package more than our actual code which is probably unnecessary. Especially since it adds ~20 more tests

Reply:
I think its safe to remove along with `MVP_SUPPORTED_COUNTRIES`